exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/json/StoredConfigurationGsonSerializerTest.java

```diff
@@ -30,6 +32,35 @@
 
 class StoredConfigurationGsonSerializerTest {
 
+  private static final String CONFIG_EXAMPLE = "{\t\n"
```

Comment:
That's one good reason to migrate to Java 12 :-) But common will probably have to stay on 8.